gazebo_dev/cmake/gazebo_dev-extras.cmake

```diff
@@ -3,11 +3,40 @@ find_package(gazebo REQUIRED)
 
 message(STATUS "Gazebo version: ${GAZEBO_VERSION}")
 
+# Workaround for https://github.com/ros-simulation/gazebo_ros_pkgs/issues/1372
+# Real fix should go into Gazebo11 https://github.com/osrf/gazebo/issues/3008
+# On Ubuntu Jammy Gazebo11 is used directly from Ubuntu, before removing this
+# please be sure that the package is updated there.
+pkg_check_modules(TBB tbb)
+set (TBB_PKG_CONFIG "tbb")
+if (NOT TBB_FOUND)
+  message(STATUS "TBB not found, attempting to detect manually")
+  set (TBB_PKG_CONFIG "")
+
+  # Workaround for CMake bug https://gitlab.kitware.com/cmake/cmake/issues/171
```

Comment:
The link the cmake issue results in a 404. Maybe the issue number is incorrect?

Reply:
it is not because I don't know how to copy/paste things. [c3556ca](https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1373/commits/c3556ca53f0206e4681cdd279225828670c15a91)